resource/Makefile

```diff
@@ -41,7 +41,7 @@ all: $(GOBINDATA_TARGET)
 debug: $(GOBINDATA_DEBUG_TARGET)
 	rm -f $(GOBINDATA_TARGET)
 
-$(TYPESCRIPT_TARGET): $(TYPESCRIPT_SOURCES)
```

Comment:
As it was, TYPESCRIPT_TARGET (app.js) was only being rebuilt if the actual directory ui/ts changed somehow.  This change makes it so that app.js is rebuilt if any file under the ts directory changes.